### DISCUSSION TITLE
Merge trivy-kubernetes into trivy

### DISCUSSION THREAD

**Discussion by josedonizetti** [aquasecurity/trivy#4388] [Ideas]
Considering the recent task to move fanal into Trivy, I wonder, wouldn't it make more sense to move trivy-kubernetes also into trivy?

- https://github.com/aquasecurity/trivy/issues/2277
- https://github.com/aquasecurity/trivy-kubernetes

@chen-keinan @knqyf263 @itaysk @liamg wdyt?

**Comment by knqyf263** [comment]
I have no objections to it.

**Comment by redsigma** [comment]
this might be helpful

https://github.com/aquasecurity/trivy-plugin-kubectl/issues/8
